config/initializers/backtrace_silencers.rb

```diff
@@ -0,0 +1,7 @@
+# Be sure to restart your server when you modify this file.
+
+# You can add backtrace silencers for libraries that you're using but don't wish to see in your backtraces.
+# Rails.backtrace_cleaner.add_silencer { |line| line =~ /my_noisy_library/ }
+
+# You can also remove all the silencers if you're trying to debug a problem that might stem from framework code.
```

Comment:
__[rubocop]__
* Line is too long. [112/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->